build.sbt

```diff
@@ -383,6 +383,7 @@ lazy val mtags3 = project
     Compile / unmanagedSourceDirectories += (ThisBuild / baseDirectory).value / "mtags-shared" / "src" / "main" / "scala-3",
     moduleName := "mtags3",
     scalaVersion := V.scala3,
+    crossScalaVersions := V.scala3Versions,
```

Comment:
We don't actually want cross version here as this is only for the IDE. We don't publish this and the other module